### DISCUSSION TITLE
Is there a PDF compilation of the documentation/tutorial?

### DISCUSSION THREAD

**Discussion by wilfredom** [ibis-project/ibis#9239] [Q&A] [answered]
Hi,

I'm really keen to learn ibis but for now I just wanted to grab some PDFs (tutorials. complete documentation) to easily search/reference and read on the move. Is there a compilation somewhere?

Thanks!!

**Comment by cpcloud** [comment, marked as answer]
We're using Quarto for documentation, and unfortunately [this isn't (yet?) possible](https://github.com/quarto-dev/quarto-cli/discussions/5782).
